src/components/HeroHeader.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { withClient } from 'cozy-client'
+import Button from 'cozy-ui/react/Button'
+import get from 'lodash/get'
+import { withBreakpoints } from 'cozy-ui/transpiled/react'
+import { translate } from 'cozy-ui/react/I18n'
+import useInstanceSettings from 'hooks/useInstanceSettings'
+import useCustomWallpaper from 'hooks/useCustomWallpaper'
+
+const HeroHeader = ({ t, client, breakpoints: { isMobile } }) => {
+  const {
+    fetchStatus,
+    data: { wallpaperLink }
+  } = useCustomWallpaper(client)
+  const rootURL = client.getStackClient().uri
+  const { host } = new URL(rootURL)
+  let backgroundURL = null
+  if (fetchStatus !== 'loading')
+    backgroundURL =
+      wallpaperLink || `${rootURL}/assets/images/default-wallpaper.jpg`
```

Comment:
Since we have a `config/collect.json` file, should we put the default path in it?